Model/Api/RouteInsuranceManagement.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2017-2022 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Model\Api;
+
+use Bolt\Boltpay\Helper\Bugsnag;
+use Exception;
+use Magento\Framework\Webapi\Exception as WebapiException;
+use Magento\Framework\Webapi\Rest\Response;
+use Magento\Framework\Module\Manager;
+use Magento\Framework\Serialize\SerializerInterface as Serialize;
+use Bolt\Boltpay\Helper\Cart as CartHelper;
+use Magento\Quote\Model\QuoteRepository;
+
+class RouteInsuranceManagement implements \Bolt\Boltpay\Api\RouteInsuranceManagementInterface
```

Comment:
Another option: use the existing `interface CartManagementInterface` and have all operations under quotes inside of the interface